aws/table_aws_ssm_maintenance_window.go

```diff
@@ -0,0 +1,343 @@
+			Hydrate: listAwsSSMMaintenanceWindow,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the  Maintenance Window.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "window_id",
+				Description: "The ID of the Maintenance Window.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "enabled",
+				Description: "Indicates whether the Maintenance Window is enabled.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "allow_unassociated_targets",
+				Description: "Whether targets must be registered with the Maintenance Window before tasks can be defined for those targets.",
+				Type:        proto.ColumnType_BOOL,
+				Hydrate:     getAwsSSMMaintenanceWindow,
+			},
+			{
+				Name:        "description",
+				Description: "A description of the Maintenance Window.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags assigned to the Maintenance Window",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsSSMMaintenanceWindowTags,
+				Transform:   transform.FromField("TagList"),
+			},
+			{
+				Name:        "duration",
+				Description: "The duration of the Maintenance Window in hours.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "cutoff",
+				Description: "The number of hours before the end of the Maintenance Window that Systems Manager stops scheduling new tasks for execution.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "schedule",
+				Description: "The schedule of the Maintenance Window in the form of a cron or rate expression.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "schedule_offset",
+				Description: "The number of days to wait to run a Maintenance Window after the scheduled CRON expression date and time.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "targets",
+				Description: "The targets of Maintenance Window.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getMaintenanceWindowTargets,
+				Transform:   transform.FromField("Targets"),
+			},
+			{
+				Name:        "tasks",
+				Description: "The Tasks of Maintenance Window.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getMaintenanceWindowTasks,
+				Transform:   transform.FromField("Tasks"),
+			},
+			{
+				Name:        "modified_date",
+				Description: "The date the Maintenance Window was last modified.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Hydrate:     getAwsSSMMaintenanceWindow,
+			},
+			{
+				Name:        "next_execution_time",
+				Description: "The next time the maintenance window will actually run, taking into account any specified times for the Maintenance Window to become active or inactive.",
+				Type:        proto.ColumnType_DATETIME,
+			},
+
```

Comment:
Can we add below columns?
```
created_date
end_date
schedule_timezone
start_date
```

Reply:
![image](https://user-images.githubusercontent.com/38218418/110131836-962bad80-7df0-11eb-8b7c-f4fb38bfae6b.png)